### PR TITLE
Lex_Prime gentx + peers

### PR DESCRIPTION
**Pull request by LexPrime**
Moniker - Lex_Prime
Twitter - https://twitter.com/Lex__Prime
Discord - Lex Prime#9111
Validator - neutronvaloper1s5fkmr2h2n5d8mnf03f4qu64ye6cl26dezua4n

### CODE REVIEW

File: quark/gentxs/Lex_Prime-gentx.json [hunk 1]
```diff
@@ -0,0 +1 @@
...1beta1.MsgCreateValidator","description":{"moniker":"Lex_Prime","identity":"0A7562DD5E1432EF","website":"https://t.me/lexxxprime","security_contact":"lex_prime@protonmail.com","details":"Darksiders Staking team validator"},"commission":{"rate":"0.050000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.100000000000000000"},"min_self_delegation":"1","delegator_address":"neutron1s5fkmr2h2n5d8mnf03f4qu64ye6cl26drl9ynh","validator_address":"neutronvaloper1s5fkmr2h2n5d8mnf03f4qu64ye6cl26dezua4n","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"VffjMkV68L4hQZ1NHHlSahrgd3DcikbY0Z4MIgPYK+Q="},"value":{"denom":"untrn","amount":"1000000000"}}],"memo":"2dd7799e27a2514c3d0cf02060a0f9b81ce33bb3@141.94.139.233:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A6y7zK/YGtet8a/e8ePp33yIqq1h67jf4is2aeisW+3v"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["FyyVIzAuIQMVaPY2juv8Bp7TwDG9KdWcrkuhuc3jZsdR4Z4loCbizA8zU/b5auwQTr57e60eNWfqvL5NmE2y2Q=="]}
```
Comment (albertandrejev) [inline quark/gentxs/Lex_Prime-gentx.json:1]: Please update `max_change_rate` in accordance with the documentation.

**Review by albertandrejev — CHANGES_REQUESTED** [review]

File: quark/gentxs/Lex_Prime-gentx.json (hunk 1 above)
Comment (LexPrime) [reply quark/gentxs/Lex_Prime-gentx.json]: Updated

File: quark/gentxs/Lex_Prime-gentx.json [hunk 2]
```diff
@@ -0,0 +1 @@
...1beta1.MsgCreateValidator","description":{"moniker":"Lex_Prime","identity":"0A7562DD5E1432EF","website":"https://t.me/lexxxprime","security_contact":"lex_prime@protonmail.com","details":"Darksiders Staking team validator"},"commission":{"rate":"0.050000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"neutron1s5fkmr2h2n5d8mnf03f4qu64ye6cl26drl9ynh","validator_address":"neutronvaloper1s5fkmr2h2n5d8mnf03f4qu64ye6cl26dezua4n","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"VffjMkV68L4hQZ1NHHlSahrgd3DcikbY0Z4MIgPYK+Q="},"value":{"denom":"untrn","amount":"1000000000"}}],"memo":"2dd7799e27a2514c3d0cf02060a0f9b81ce33bb3@141.94.139.233:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A6y7zK/YGtet8a/e8ePp33yIqq1h67jf4is2aeisW+3v"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["FyyVIzAuIQMVaPY2juv8Bp7TwDG9KdWcrkuhuc3jZsdR4Z4loCbizA8zU/b5auwQTr57e60eNWfqvL5NmE2y2Q=="]}
```
Comment (albertandrejev) [inline quark/gentxs/Lex_Prime-gentx.json:1]: Sorry, you need to recreate gentx with right parameter using command, it is not possible to update gentx content manually, it corrupts file.

**Review by albertandrejev — CHANGES_REQUESTED** [review]

File: quark/gentxs/Lex_Prime-gentx.json (hunk 2 above)
Comment (LexPrime) [reply quark/gentxs/Lex_Prime-gentx.json]: Ok, gentx recreated

**Review by albertandrejev — APPROVED** [review]

**Pull request merged**